astrocut/cube_cut.py

```diff
@@ -668,15 +672,12 @@ def _apply_header_inherit(self, hdu_list):
                         hdu.header[kwd] = (primary_header[kwd], primary_header.comments[kwd])
             
 
-    def _build_tpf(self, product, cube_fits, img_cube, uncert_cube, cutout_wcs_dict, aperture):
+    def _build_tpf(self, cube_fits, img_cube, cutout_wcs_dict, aperture, uncert_cube=None):
```

Comment:
if when we make the cutout, it returns `None` for `uncert_cube`, we can call this exactly the same for both TICA and SPOC and we can leave uncert_cube as a required param. then we don't need another conditional when we call this function, since for TICA `None` will be passed in.